low-level/framework/utils/consulstore.py

```diff
@@ -104,10 +104,37 @@ def get(self, key, **kwargs):
     def exists(self, key):
         """check if key exists
         """
-        if self.get(key):
-            return True
-        else:
-            return False
+        data = None
```

Comment:
We don't have any way to differentiate exists method has returns None due to connection error or path not exists.
And Internally it uses Get method of the ConsulStore. I re-wrote exists API to avoid changes in the Get method at this stage as it used more then 20 files. We can improve later and add error handling in the all the API of the ConsulStore.

Reply:
Agree